### ISSUE TITLE
Add support for anvironment variables

### ISSUE THREAD

**Issue by Kostanos** [MattiLehtinen/postgrator-cli#7]
Instead of configuration file or parameters, would be great if postgrator-cli supports environment variables, for example, for default "pg" engine they are PGUSER, PGHOST, PGPASSWORD, PGDATABASE
This will simplify a lot in multidocker environments.

**Comment by madflow** [comment]
I believe it is already possible to use env variables - when you use a `.js` file instead of `.json`:

```JS
// postgrator.js
const url = require('url');

let host; let port; let database; let username; let password;
if (process.env.DATABASE_URL) {
    const dbUrl = url.parse(process.env.DATABASE_URL);
    host = dbUrl.hostname;
    port = dbUrl.port;
    database = dbUrl.pathname.replace(/^\//, '');
    [username, password] = dbUrl.auth.split(':');
}

module.exports = {
    migrationDirectory: 'test/migrations',
    driver: 'pg',
    host,
    port,
    database,
    username,
    password,
};
```
```bash
$ postgrator -c postgrator.js 
```

**Comment by perrin4869** [comment]
It is probably irrelevant by now, but this kind of usecase is supported in my fork [`perrin4869/postgrator-cli`](https://www.npmjs.com/package/@perrin4869/postgrator-cli) - see [here](https://github.com/perrin4869/postgrator-cli/blob/c11efba72e4a603ae1feca290cc9849aad9243bc/test/postgrator-cli-tests.js#L240).

**Comment by perrin4869** [comment]
Fixed in #30
